splink/dialects.py

```diff
@@ -50,6 +64,10 @@ def name(self):
     def levenshtein_function_name(self):
         return "levenshtein"
 
+    @property
+    def jaro_winkler_function_name(self):
+        return "jaro_winkler"
```

Comment:
Technically there isn't a jaro winkler on sqllite, the user would have to register it as a custom python udf.  I'm not sure what the best solution is, but for now Id probably not include this here - the user could use a (forthcoming) generic distance level

Reply:
Oh wait,my bad, we automatically register it for the user so we can keep this:
https://github.com/moj-analytical-services/splink/blob/868fefba022483f4cb0d05822aa55d62ee9b92fa/splink/sqlite/linker.py#L227

Reply:
It might be worth putting a comment to note that's what's happening?

Reply:
atm we have a default (opt-out) of using registering udfs backed by `rapidfuzz` when we instantiate a `SQLiteLinker` - see [a note in the docs](https://moj-analytical-services.github.io/splink/topic_guides/splink_fundamentals/backends/backends.html#sqlite_1) and the [relevant linker code](https://github.com/moj-analytical-services/splink/blob/868fefba022483f4cb0d05822aa55d62ee9b92fa/splink/sqlite/linker.py#L223)

Reply:
I've added an additional note to the dialect to make this clear.